prep-javascript-objects/main.js

```diff
@@ -0,0 +1,17 @@
+const person = {
+  firstName: 'Steve',
+  lastName: 'Martin',
+  hobby: 'banjo playing'
+};
+
+console.log(person);
+console.log("The person's name is " + person.firstName + ' ' + person.lastName + '.');
```

Comment:
Here you want to first create a variable to hold the persons `fullName` then use the variable in your console log for the full name.